common/test/acceptance/tests/studio/test_studio_help.py

```diff
@@ -47,3 +63,357 @@ def test_studio_help_links(self):
                 expected['sr_text'],
                 actual.find_element_by_xpath('following-sibling::span').text
             )
+
+
+class SignInAndSignUpHelpTest(WebAppTest):
+    """
+    Tests help links on 'Sign In' and 'Sign Up' pages.
+    """
+    def setUp(self):
+        super(SignInAndSignUpHelpTest, self).setUp()
+        self.index_page = IndexPage(self.browser)
+
+    def test_help_links_on_signup_and_sign_in_pages(self):
+        """
+        Scenario: Help link in navigation bar is working on
+        'Sign Up' and 'Sign In' page.
+        Given that I am on the 'Sign Up' or 'Sign In' page.
+        And I click the 'Help' in the navigation bar
+        Then Help link should open.
+        And help url should contain 'getting_started/get_started.html'
+
+        """
+        expected_links = [
+            {
+                'element_css': '.nav-item.nav-not-signedin-help a',
+                'text': 'Help',
+                'url': 'http://edx.readthedocs.org/projects/edx-partner-course-staff'
+                       '/en/latest/getting_started/get_started.html',
+                'page': 'index-sign-in'
+            },
+            {
+                'element_css': '.nav-item.nav-not-signedin-help a',
+                'text': 'Help',
+                'url': 'http://edx.readthedocs.org/projects/edx-partner-course-staff/'
+                       'en/latest/getting_started/get_started.html',
+                'page': 'index-sign-up'
+            }
+        ]
```

Comment:
@Muddasser Best practice is to have css selectors in the page object file. Test file should only have steps to test different scenarios. Also try to move other things into page object file if possible.